docs/src/app/navigationConstants.js

```diff
@@ -282,6 +296,59 @@ export const navigationItems = [
                     {
                         component: 'page',
                         label: 'Page',
+                        levelFour: [
+                            {
+                                component: 'pageDemo',
+                                emit: true,
```

Comment:
Is `emit` supposed to maybe be [`omit`](https://www.merriam-webster.com/dictionary/omit), is in "omit this one!  skip it!"

Reply:
Fixed.

Reply:
Yeah, that makes more sense.  Thanks for changing that.  Resolving.